javascript/components/FormInput.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class ContactField extends Component {
```

Comment:
I'm curious why you decided to put all the various inputs into a single component class, rather than have, e.g. `<TextInput>`, `<Dropdown>`, etc.? It doesn't seem like the component is really doing anything common to all the various types, other than perhaps the `<label>` element.

Reply:
Good point. There really isn't as much in common as I initially thought.